lib/VisualBuilder.js

```diff
@@ -34,6 +34,7 @@ let Validator = require('jsonschema').Validator;
 let TypescriptCompiler = require('../lib/TypescriptCompiler');
 let LessCompiler = require('../lib/LessCompiler');
 let config = require('../config.json');
+let ConsoleWriter = require('../lib/ConsoleWriter');
```

Comment:
I'm trying to only do console output from files in the "bin" folder which is the actual CLI app. 

I'd like this to be able to one day be used as library by other 3rd party apps (possibly something like a gulp task that could build visuals) therefore I want things in the lib folder to not output to console.